src/providers/emoji/Emojis.cpp

```diff
@@ -296,31 +319,61 @@ std::vector<boost::variant<EmotePtr, QString>> Emojis::parse(
 
         for (const std::shared_ptr<EmojiData> &emoji : possibleEmojis)
         {
+            int emojiNonQualifiedExtraCharacters =
```

Comment:
This and all other `int`s here should be `QString::size_type` (can often use `auto`) for compatibility with both Qt 5 and Qt 6.

Reply:
Fixed in [`1213173` (#4840)](https://github.com/Chatterino/chatterino2/pull/4840/commits/1213173a88fc62a174c42253c3efcabc971ff50b)